config/locales/en.yml

```diff
@@ -1293,6 +1294,24 @@ en:
       will_be_moved: "%{team_name}'s users and open cases will be moved to %{target_directorate_name}"
     team: Business unit
     unreversible: This action cannot be reversed
+  directorates:
+    move:
+      confirm_heading: Confirm move
+      heading: Move directorate
+      intro:
+        bullet1: all business unites will move with it
+        bullet2: all open cases and users under each business unit will move with it
+        bullet3: all members under each business unit will be able to view closed cases
+        bullet4: performance reports for the unit both before and after the move are available
+        paragraph: "Moving a directorate means:"
+      moved_successfully: "%{team_name} has been moved to %{destination_business_group_name}"
+      moving_to: Moving to
+      subhead: "Directorate: %{team_name}"
+      team_currently_located: This is where the team is currently located
+      will_be_moved: "The users and open cases assigned to the business units under %{team_name} will be moved to %{target_business_group_name}"
+      error: "Failed to move this directorate due to reason: #{reason}"
+    unreversible: This action cannot be reversed
```

Comment:
Was wondering how true this is if I can just move it back to the one before

Reply:
I think the 'cannot be reversed' here is more from technique point of view, means we cannot simply undo the movement action as there are lots of records which will be created and updated on database level, and those changes actually affect those open cases.

Reply:
OK, thanks